lib/v2/screens/settings/edit_name/components/text_form_field_custom.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
```

Comment:
do we need a custom input field? 
Is the basic one too different from designs? 

We might need to talk to Jason and ask him to use flutter default input fields, I dont like creating custom widgets unless absolutely needed

Reply:
Unless we want to repeat the decorator code in each new TextForm and other pre sets behaivors

Reply:
My point is, If we use the default one, we dont need decoration. Or just very very basic decoration. I would rather no decoration at all. 

Is the default one too differnet from designs ?

Reply:
I am very sure that the default is way different from how the required desing looks, ​​but in the afternoon we can chat the differences well.

Reply:
lets not create a new `TextFormFieldCustom` use the default one

Reply:
Okay